### ISSUE TITLE
Party invitations via email not working as described

### ISSUE THREAD

**Issue by shanaqui** [HabitRPG/habitica#10479]
People in the Bug Guild have been reporting that when they send party invitations via email, it sends an email which prompts people to sign up, but once they sign up, there's no invitation in their notifications as we suggest there will be.

**Comment by Alys** [comment]
We had this issue a while back: https://github.com/HabitRPG/habitica/issues/9450
Maybe the discussions in that issue and the linked PR are still relevant.

**Comment by probaton** [comment]
I have absolutely no idea how invitation processing is managed in Habitica, but it sounds like an interesting challenge. Sign me up.

**Comment by probaton** [comment]
I did some tinkering and it looks like the login page overrides the URL suffix to `/static/home` while it's loading, which erases the query string in the URL. 

The chain for sending new users a party invite is triggered in `register()` in `home.vue`, which bases the decision to send invites on the presence of a `p`/`groupInvite` option in the query string. No query string means no invites. 

I clicked around for a bit, but as my domain knowledge on routing is sorely lacking, I have no idea where this behavior might be governed. I don't mind putting in some more time, but I'm going to need someone to point me in the right direction. Or you're going to have to find someone who knows what they're doing.
